docs/en/latest/tutorials/cache-api-responses.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Cache API responses
+keywords:
+  - API Gateway
+  - Apache APISIX
+  - Cache
+  - Performance
+description: This tutorial will focus primarily on handling caching at the API Gateway level by using Apache APISIX API Gateway and you will learn how to use proxy-caching plugin to improve response efficiency for your Web or Microservices API.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This tutorial will focus primarily on handling caching at the API Gateway level by using Apache APISIX API Gateway and you will learn how to use proxy-caching plugin to improve response efficiency for your Web or Microservices API.
+
+**Here is an overview of what we cover in this walkthrough:**
+
+- ✔️ Caching in API Gateway
+- ✔️ About [Apache APISIX API Gateway](https://apisix.apache.org/docs/apisix/getting-started/)
+- ✔️ Run the demo project [apisix-dotnet-docker](https://github.com/Boburmirzo/apisix-dotnet-docker)
+- ✔️ Configure the [Proxy Cache](https://apisix.apache.org/docs/apisix/plugins/proxy-cache/) plugin
+- ✔️ Validate Proxy Caching
+
+## Improve performance with caching
+
+When you are building an API, you want to keep it simple and fast. Once the concurrent need to read the same data increase, you'll face a few issues 😐 where you might be considering introducing **caching**:
+
+- ❌ There is latency on some API requests which is noticeably affecting the user's experience.
+- ❌ Fetching data from a database takes more time to respond.
+- ❌ Availability of your API is threatened by the API's high throughput.
+- ❌ There are some network failures in getting frequently accessed information from your API.
```

Comment:
I recommend removing these special symbols in the tutorial documentation.

Reply:
@hf400159 should this point be added to our documentation style guide? https://apisix.apache.org/docs/general/documentation-style-guide/

Reply:
@hf400159 Good idea!